addons/event/data/event_demo.xml

```diff
@@ -146,6 +146,7 @@
         <field name="date_end" eval="(DateTime.today()+ timedelta(days=133)).strftime('%Y-%m-%d 00:30:00')"/>
         <field name="event_type_id" ref="event_type_0"/>
         <field name="address_id" ref="event.res_partner_location_1"/>
+        <field name="date_tz">Europe/London</field>
```

Comment:
Shouldn't we synchronize somehow address_id.tz and event date_tz ? If we set an address we probably want to take its timezone if set on the partner ... what do you think ?